docs/user/environment/0-setup/elevation.rst

```diff
@@ -0,0 +1,66 @@
+Elevation
+=========
+
+The elevation of the launch site of a rocket is particularly important for
+determining its interaction with the atmosphere and Earth's gravitational field.
+
+With previously known elevation
+-------------------------------
+
+If the elevation, measured in meters above sea level, along with the latitude
+and longitude of the launch site is known, it can be used to initialize an
+Environment class instance as follows:
```

Comment:
At first glance, it seems that the latitude and longitude must be known for almost all methods that set up an environment. If that is indeed the case, maybe it would be relevant to comment on that in the previous subsection. For instance, we could write in the previous paragraph:

"... and Earth's gravitational field. In most cases, the latitude and longitude must be set together with the elevation."

and on this section:

"If the elevation, measured in meters above sea level, is known, ..."